src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,76 @@
+(function(transport, global, console) {
+    "use strict";
+    
+    function transportSend(args, type) {        
+        // To avoid Runtime.evaluate messages
+        if(args[0] !== "Runtime.evaluate") {
+            transport.send("bramble-console", args, type);
+        }
+    }
+    
+    function generateRouter(fname) {
+        var args = Array.from(arguments).slice();
+        transportSend(args, fname);
+    }
+    
+    // Implement console.log replacement
+    console.log = function() {
+        var args = Array.from(arguments).slice();
+        transportSend(args, "log");
+    };
+    
+    // Implement console.debug replacement
+    console.debug = function() {
+        var args = Array.from(arguments).slice();
+        transportSend(args, "debug");
+    };
+    
+    // Implement console.error replacement
+    console.error = function() {
+        var args = Array.from(arguments).slice();
+        transportSend(args, "error");
+    };
+    
+    // Implement console.info replacement
+    console.info = function() {
+        var args = Array.from(arguments).slice();
+        transportSend(args, "info");
+    };
+    
+    // Implement console.warn replacement
+    console.warn = function() {
+        var args = Array.from(arguments).slice();
+        transportSend(args, "warn");
+    };
+    
+    // Implement console.clear replacement
+    console.clear = function() {
+        var args = Array.from(arguments).slice();
+        transportSend(args, "clear");
+    };
+    
+    // Implement console.time replacement
```

Comment:
For `time, timeEnd, assert` which are not using the same boilerplate, you can do this no problem.  However, please deal with your custom arg processing for `time, endTime` in here.